src/schema/event/resolvers.ts

```diff
@@ -22,11 +23,16 @@ const Query: QueryResolvers = {
       meta: data.meta,
     };
   },
-  eventsByIds: async (_, { ids, include }, { dataSources }) => {
+  eventsByIds: async (_, { ids, include, source }, { dataSources }) => {
+    // TEMPORARY HACK (linkedcourses doesn't suport new way of getting list of ids!!)
```

Comment:
Have we asked linked team to fix it?

Reply:
Asked Daniel about this but still waiting for answer

Reply:
This can be done later